homeassistant/components/homeassistant/services.yaml

```diff
@@ -13,6 +13,11 @@ set_location:
       example: 117.22743
       selector:
         text:
+    elevation:
+      required: false
+      example: 120
+      selector:
+        text:
```

Comment:
Shouldn't we use a number selector in `box` mode with step `any`?

https://www.home-assistant.io/docs/blueprint/selectors/#number-selector

Same goes for the latitude and longitude.

Reply:
See https://github.com/home-assistant/core/pull/100360